newrelic/resource_newrelic_cloud_azure_link_account.go

```diff
@@ -147,38 +149,40 @@ func resourceNewRelicCloudAzureLinkAccountRead(ctx context.Context, d *schema.Re
 func readAzureLinkedAccount(d *schema.ResourceData, result *cloud.CloudLinkedAccount) {
 	_ = d.Set("account_id", result.NrAccountId)
 	_ = d.Set("name", result.Name)
+	_ = d.Set("application_id", result.AuthLabel)
+	_ = d.Set("disabled", result.Disabled)
+	_ = d.Set("subscription_id", result.ExternalId)
```

Comment:
@vagrawal-newrelic everything else looks good, just a couple of testing questions linked to this block
- upon creating a azure link account (applying it) without these changes, then compiling these changes locally and running a plan, do you see any drift caused by `disabled`? If not, we should be good to go 
- upon creating a azure link account (applying it) without these changes, then compiling these changes locally and running a plan, do you see any drift caused by other attributes because they are now being saved to the state? Ideally, this should not be the case; if it is not so, I think we should be good to add `tenant_id` too here (did you not want to add it here since an update of the `tenant_id` is not allowed by the API? Just trying to understand :) ). With these checks, we should be good to go
- can you please make sure `application_id`, `subscription_id` (and `tenant_id` if added) are saved to the state as obfuscated (masked with `***`) values? If so, we should be good to go

Reply:
upon creating a azure link account (applying it) without these changes, then compiling these changes locally and running a plan,  **observaltion**: `No drift found`
`Please see the attached doc for the validation document`
[azure_link_account_update.pdf](https://github.com/user-attachments/files/17538655/azure_link_account_update.pdf)


`tenant_id` is not being considered above beacause the NerdGraph query is not. returning tenantID in the response. Also update mutation response is not returning tenantID.

<img width="1012" alt="image" src="https://github.com/user-attachments/assets/84a69e94-5c8f-4608-9bcc-b314b7739c0e">


Also Please note that disabled field is available only for update mutation not for create mutation. same have been mentioned in the documentation. 
<img width="351" alt="image" src="https://github.com/user-attachments/assets/d160f295-c95e-4fcf-bf37-69cd1a067199">